### PR TITLE
update public workbench api version to 47.0

### PR DESCRIPTION
**Pull request by amihaihalevy**

### CODE REVIEW

**Comment by amihaihalevy** [comment]
@john-brock @ntotten @vazexqi can one of you review and merge this one?

**Comment by amihaihalevy** [comment]
@john-brock @ntotten @vazexqi can you help here? if not, can you refer me to the person who is responsible for this repo?

Thanks!

File: workbench/config/constants.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 //
 //////////////////////////////////////////////////////////////////////////////////////////
 
-$GLOBALS["WORKBENCH_VERSION"] = "46.0.0";
+$GLOBALS["WORKBENCH_VERSION"] = "47.0.0";
```
Comment (ntotten) [inline workbench/config/constants.php]: Default version should not be changed until R2

**Comment by amihaihalevy** [comment]
@ntotten ok. I'll wait after 222 RF.

**Comment by RoyGilad** [comment]
Amihai
You could make an local version of workbench, and to keep working with v47
on your computer.

**Pull request merged**